app/components/MovieDetail/MovieDetail.jsx

```diff
@@ -2,12 +2,17 @@ import './_MovieDetail.scss';
 import React from 'react';
 import _ from 'lodash';
 import MovieStore from '../../stores/MovieStore';
+import AppActions from '../../actions/AppActions';
 
 const findMovieFromAll = (props) => {
 	let title = _.get(props.context, 'params.title', '');
 	return MovieStore.getByTitle(title);
 }
 
+const addToCart = (i, j) => {
+	AppActions.addMovieToCart(i.context);
```

Comment:
I don't think the cart needs the context. It should either take teh movie itself from `findMovieFromAll` or even better just the title to look it up later